linux/videodev.h has moved to libv4l-dev and changed its name.
Index: raydium-0.0svn20100207/raydium/live.h
===================================================================
--- raydium-0.0svn20100207.orig/raydium/live.h	2010-02-07 02:30:01.000000000 +0100
+++ raydium-0.0svn20100207/raydium/live.h	2012-10-13 20:22:17.252355886 +0200
@@ -16,7 +16,7 @@
 #ifndef WIN32
 #ifndef APPLE
 #include <linux/types.h>
-#include <linux/videodev.h>
+#include <libv4l1-videodev.h>
 #include <sys/mman.h>
 #endif
 #else
